src/mysky/index.ts

```diff
@@ -166,6 +166,20 @@ export class MySky {
   // Public API
   // ==========
 
+  /**
+   * Checks if the current browser is supported by MySky.
+   *
+   * @returns - A promise with a boolean indicating whether the browser is supported and, if not, a string containing the user-friendly error message.
+   */
+  static async isBrowserSupported(): Promise<[boolean, string]> {
```

Comment:
`nit` little redundant to return` a promise with two vars here, maybe just `Promise<Error|null>` but I guess if you're using the `Promise<[boolean, string]>` pattern it's fine

Reply:
Okay, I'm not too familiar with either pattern but I'm keeping this since you said it's fine. 😂